### PR TITLE
BO-1813: IG for viewing and partially editing abstract products.

### PR DESCRIPTION
**Pull request by tobi812**
## PR Description

IG for viewing and partially editing abstract products.

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md [hunk 1]
```diff
@@ -36,10 +36,52 @@ Make sure that the following modules have been installed:
 |-|-|
 | ProductMerchantPortalGui | spryker/product-merchant-portal-gui |
 | ProductMerchantPortalGuiExtension | spryker/product-merchant-portal-gui-extension |
+| AvailabilityMerchantPortalGui | spryker/availability-merchant-portal-gui |
+| ProductOfferMerchantPortalGui | spryker/product-offer-merchant-portal-gui |
+| MerchantStockDataImport | spryker/merchant-stock-data-import |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md:41]: why you need it?

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md [hunk 2]
```diff
@@ -36,10 +36,52 @@ Make sure that the following modules have been installed:
 |-|-|
 | ProductMerchantPortalGui | spryker/product-merchant-portal-gui |
 | ProductMerchantPortalGuiExtension | spryker/product-merchant-portal-gui-extension |
+| AvailabilityMerchantPortalGui | spryker/availability-merchant-portal-gui |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md:39]: Why? This module doesn't belong to this feature. We should have an additional IG - `Merchant Portal - Marketplace Product + Inventory Management feature integration` where spryker/availability-merchant-portal-gui will be added and the plugin registered. 
See `Merchant Portal - Marketplace Product + Tax feature integration` as example.
See https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides - `Optional feature` section where is described how to build such IGs

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-inventory-management-feature-integration.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+title: Merchant Portal - Marketplace Product + Inventory Management feature integration
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-inventory-management-feature-integration.md:2]: please add to navigation

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md [hunk 4]
```diff
@@ -54,7 +55,10 @@ Make sure that the following changes have been applied in transfer objects:
 | TRANSFER  | TYPE  | EVENT | PATH  |
 |-|-|-|-|
 | MerchantProductTableCriteria | class | Created | src/Generated/Shared/Transfer/MerchantProductTableCriteriaTransfer |
+| MerchantProductCriteria.idMerchant | property | Created | src/Generated/Shared/Transfer/MerchantProductCriteriaTransfer |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md:58]: not needed, already specified in Merchant product feature IG

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md [hunk 5]
```diff
@@ -54,7 +55,10 @@ Make sure that the following changes have been applied in transfer objects:
...
 | ProductAbstractCollection | class | Created | src/Generated/Shared/Transfer/ProductAbstractCollectionTransfer |
+| ProductTableCriteria | class | Created | src/Generated/Shared/Transfer/ProductTableCriteriaTransfer |
+| CriteriaRangeFilter | class | Created | src/Generated/Shared/Transfer/CriteriaRangeFilterTransfer |
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-product-feature-integration.md:61]: not needed, already specified in Marketplace core feature IG (exists not is this branch but in master)

**Review by ilyakubanov — DISMISSED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**